src/server/container.ts

```diff
@@ -49,6 +49,7 @@ export default class Container {
     private readonly startTime = process.hrtime();
     private pty: ptylib.IPty | null = null;
     private exited = false;
+    private cols: number = null;
```

Comment:
Can we rename this to `terminalWidth` so that it's clearer what this means in this context (especially since there's no corresponding "rows" variable here)?